apps/www/_blog/2024-05-01-pgvector-0-7-0.mdx

```diff
@@ -0,0 +1,184 @@
+---
+title: "What's new in pgvector v0.7.0"
+description: 'Exploring new features in pgvector v0.7.0'
+author: pavel
+image: pgvector-0-7-0.png
+thumb: pgvector-0-7-0.png
+categories:
+  - engineering
+tags:
+  - supabase-engineering
+  - planetpg
+date: '2024-05-01'
+toc_depth: 3
+---
+
+### New vector types
+
+Real-world embedding datasets contain a lot of redundancy. It’s buried in the vector space. For example, when vectors are clustered around some central points in multidimensional, we’re seeing exploitable structure. Reducing this makes it possible to get memory and performance savings with no effect on precision. There are several different approaches to employ this ideas made available in pgvector since 0.7.0:
+
+- float16 vector representation
+- sparse vectors
+- bit vectors
+
+### Float16 vectors
+
+An HNSW index is are most efficient when the index fits into shared memory and is not evicted due to concurrent Postgres operations. The reason for this is that it allows Postgres to minimize costly IO. Historically, pgvector supported only 32-bit vectors. In version 0.7.0 pgvector introduces 16-bit float HNSW indexes which consume exactly half the memory. That reduction in memory keeps operations at maximum performance for twice as long.
```

Comment:
```suggestion
An HNSW index is most efficient when the index fits into shared memory and is not evicted due to concurrent Postgres operations. The reason for this is that it allows Postgres to minimize costly IO. Historically, pgvector supported only 32-bit vectors. In version 0.7.0 pgvector introduces 16-bit float HNSW indexes which consume exactly half the memory. That reduction in memory keeps operations at maximum performance for twice as long.
```

Reply:
Batched this in with some other suggestions